src/Owin.Antiforgery/CsrfConstants.cs

```diff
@@ -13,7 +13,7 @@ public static class CsrfConstants
         // Max-Age in seconds for the default base cookie. 365 days.
         public static readonly int MaxAge = 365 * 24 * 60 * 60;
 
-        public static readonly int TokenLength = 24;
+        public static readonly int TokenLength = 23;
```

Comment:
When encoding with the new method the length is now 23. 24 would fail otherwise.